challenges/generic-function-arguments/prompt.md

```diff
@@ -0,0 +1,80 @@
+## Why Functions Need Generics
+
+One of the most common needs a TypeScript developer faces is the need to pass generic type arguments to functions.  Then, those functions can use and manipulate those parameters.
+
+In many situations, we don't need functions to be generic.  The function might be written in such a way as to only work for one type of data (e.g. `string` or `number`).  But, other times, we write functions where we want to pass the data type along from input to output.
+
+## Understanding Generic Syntax
+
+In a previous challenge on [generic type arguments](todo-generic-type-arguments) we saw a `Row` type that looked like this:
```

Comment:
we can probably start doing `https://typehero.dev/challenge/my-slug`

Reply:
I fixed all slugs in https://github.com/typehero/typehero/pull/947/commits/f3942701e4005c43f4190d3ce4b706af0842b6b7